diff --git a/node_modules/stacktrace-parser/dist/stack-trace-parser.d.ts b/node_modules/stacktrace-parser/dist/stack-trace-parser.d.ts
index 7f0fd30..6cb0b3f 100644
--- a/node_modules/stacktrace-parser/dist/stack-trace-parser.d.ts
+++ b/node_modules/stacktrace-parser/dist/stack-trace-parser.d.ts
@@ -1,8 +1,6 @@
-import { LiteralUnion } from 'type-fest';
-
 export interface StackFrame {
   file: string | null;
-  methodName: LiteralUnion<'<unknown>', string>;
+  methodName: '<unknown>' | (string & Record<never, never>)
   arguments: string[];
   lineNumber: number | null;
   column: number | null;
