commit 8aec247d5798cb7b4f04eb58d08742506dd7495c
Author: Derrick Marcey <dmarcey@github.com>
Date:   Thu May 8 04:49:14 2025 -0700

    Include routes in new data router app

diff --git a/ui/packages/hosted-compute-ims-stafftools/hosted-compute-ims-stafftools.ts b/ui/packages/hosted-compute-ims-stafftools/hosted-compute-ims-stafftools.ts
index 822c27de5853d..9d79971e0f447 100644
--- a/ui/packages/hosted-compute-ims-stafftools/hosted-compute-ims-stafftools.ts
+++ b/ui/packages/hosted-compute-ims-stafftools/hosted-compute-ims-stafftools.ts
@@ -5,6 +5,9 @@ import {registerDataRouterApp, registerNavigatorApp} from '@github-ui/react-core
 import {jsonRoute} from '@github-ui/react-core/json-route'
 import {CuratedImageVersionDetailsPage} from './routes/CuratedImageVersionDetailsPage'
 import {hostedComputeImsStafftoolsAppBuilder} from './config/app-builder'
+import {hostedComputeImageDetailsRoute} from './routes/hosted-compute-image-details-route'
+import {hostedComputeImageVersionDetailsRoute} from './routes/hosted-compute-image-version-details-route'
+import {hostedComputeImsAdminRoute} from './routes/hosted-compute-ims-admin-route'

 registerNavigatorApp('hosted-compute-ims-stafftools', () => ({
   App,
@@ -21,5 +24,16 @@ registerNavigatorApp('hosted-compute-ims-stafftools', () => ({
   ],
 }))

-export const hostedComputeImsStafftoolsApp = hostedComputeImsStafftoolsAppBuilder.createDataRouterAppFromRoutes([])
+export const hostedComputeImsStafftoolsApp = hostedComputeImsStafftoolsAppBuilder.createDataRouterAppFromRoutes([
+  hostedComputeImsAdminRoute.toRoute({
+    Component: MainPageEntrypoint,
+  }),
+  hostedComputeImageDetailsRoute.toRoute({
+    Component: CuratedImageDetailsPageEntrypoint,
+  }),
+  hostedComputeImageVersionDetailsRoute.toRoute({
+    Component: CuratedImageVersionDetailsPageEntrypoint,
+  }),
+])
+
 registerDataRouterApp(hostedComputeImsStafftoolsApp)
+

