commit 09185cd2f997b26a1b21ef50144d06257a95ca0b
Author: Derrick Marcey <dmarcey@github.com>
Date:   Thu May 8 04:41:01 2025 -0700

    Create and register `hostedComputeImsStafftoolsApp`

diff --git a/ui/packages/hosted-compute-ims-stafftools/hosted-compute-ims-stafftools.ts b/ui/packages/hosted-compute-ims-stafftools/hosted-compute-ims-stafftools.ts
index 6d95f854fda64..822c27de5853d 100644
--- a/ui/packages/hosted-compute-ims-stafftools/hosted-compute-ims-stafftools.ts
+++ b/ui/packages/hosted-compute-ims-stafftools/hosted-compute-ims-stafftools.ts
@@ -1,9 +1,10 @@
 import {App} from './App'
 import {MainPage} from './routes/MainPage'
 import {CuratedImageDetailsPage} from './routes/CuratedImageDetailsPage'
-import {registerNavigatorApp} from '@github-ui/react-core/register-app'
+import {registerDataRouterApp, registerNavigatorApp} from '@github-ui/react-core/register-app'
 import {jsonRoute} from '@github-ui/react-core/json-route'
 import {CuratedImageVersionDetailsPage} from './routes/CuratedImageVersionDetailsPage'
+import {hostedComputeImsStafftoolsAppBuilder} from './config/app-builder'

 registerNavigatorApp('hosted-compute-ims-stafftools', () => ({
   App,
@@ -19,3 +20,6 @@ registerNavigatorApp('hosted-compute-ims-stafftools', () => ({
     }),
   ],
 }))
+
+export const hostedComputeImsStafftoolsApp = hostedComputeImsStafftoolsAppBuilder.createDataRouterAppFromRoutes([])
+registerDataRouterApp(hostedComputeImsStafftoolsApp)
+

