commit 33e76ba06088ea0549ff63f60d141a9d0e411f85
Author: Derrick Marcey <dmarcey@github.com>
Date:   Thu May 8 04:46:24 2025 -0700

    Create new routes

diff --git a/ui/packages/hosted-compute-ims-stafftools/routes/hosted-compute-image-details-route.ts b/ui/packages/hosted-compute-ims-stafftools/routes/hosted-compute-image-details-route.ts
new file mode 100644
index 0000000000000..dbec9cbaed928
--- /dev/null
+++ b/ui/packages/hosted-compute-ims-stafftools/routes/hosted-compute-image-details-route.ts
@@ -0,0 +1,15 @@
+import {mainQuery} from '@github-ui/react-core/future/main-query'
+import {hostedComputeImsStafftoolsAppBuilder} from '../config/app-builder'
+import type {CuratedImageDetailsPagePayload} from './CuratedImageDetailsPage'
+
+export const hostedComputeImageDetailsRoute = hostedComputeImsStafftoolsAppBuilder.createQueryRouteConfig(
+  'hostedComputeImageDetailsRoute',
+  {
+    path: '/stafftools/hosted_compute_ims_admin/curated_images/:image_definition_id',
+    queries: [
+      mainQuery<CuratedImageDetailsPagePayload>(),
+    ],
+  },
+)
+
diff --git a/ui/packages/hosted-compute-ims-stafftools/routes/hosted-compute-image-version-details-route.ts b/ui/packages/hosted-compute-ims-stafftools/routes/hosted-compute-image-version-details-route.ts
new file mode 100644
index 0000000000000..f1b0397205586
--- /dev/null
+++ b/ui/packages/hosted-compute-ims-stafftools/routes/hosted-compute-image-version-details-route.ts
@@ -0,0 +1,15 @@
+import {mainQuery} from '@github-ui/react-core/future/query-configs'
+import {hostedComputeImsStafftoolsAppBuilder} from '../config/app-builder'
+import type {CuratedImageVersionDetailsPagePayload} from './CuratedImageVersionDetailsPage'
+
+export const hostedComputeImageVersionDetailsRoute = hostedComputeImsStafftoolsAppBuilder.createQueryRouteConfig(
+  'hostedComputeImageVersionDetailsRoute',
+  {
+    path: '/stafftools/hosted_compute_ims_admin/curated_images/:image_definition_id/versions/:version',
+    queries: [
+      mainQuery<CuratedImageVersionDetailsPagePayload>(),
+    ],
+  },
+)
+
diff --git a/ui/packages/hosted-compute-ims-stafftools/routes/hosted-compute-ims-admin-route.ts b/ui/packages/hosted-compute-ims-stafftools/routes/hosted-compute-ims-admin-route.ts
new file mode 100644
index 0000000000000..96fdcee5c86a7
--- /dev/null
+++ b/ui/packages/hosted-compute-ims-stafftools/routes/hosted-compute-ims-admin-route.ts
@@ -0,0 +1,15 @@
+import {mainQuery} from '@github-ui/react-core/future/query-configs'
+import {hostedComputeImsStafftoolsAppBuilder} from '../config/app-builder'
+import type {MainPagePayload} from './MainPage'
+
+export const hostedComputeImsAdminRoute = hostedComputeImsStafftoolsAppBuilder.createQueryRouteConfig(
+  'hostedComputeImsAdminRoute',
+  {
+    path: '/stafftools/hosted_compute_ims_admin',
+    queries: [
+      mainQuery<MainPagePayload>(),
+    ],
+  },
+)
+

