commit 49fac9758c0e0a78a56ca73dfda4deab421e8cf6
Author: Derrick Marcey <dmarcey@github.com>
Date:   Thu May 8 04:39:41 2025 -0700

    Create app builder for `hosted-compute-ims-stafftools` package

diff --git a/ui/packages/hosted-compute-ims-stafftools/config/app-builder.ts b/ui/packages/hosted-compute-ims-stafftools/config/app-builder.ts
new file mode 100644
index 0000000000000..dd47f6b9cb2b9
--- /dev/null
+++ b/ui/packages/hosted-compute-ims-stafftools/config/app-builder.ts
@@ -0,0 +1,3 @@
+import {DataRouterApplicationBuilder} from '@github-ui/react-core/future/data-router-application'
+
+export const hostedComputeImsStafftoolsAppBuilder = DataRouterApplicationBuilder.create('hosted-compute-ims-stafftools')
+

