commit 37b8ea77c26fdc8db53232d4e9bf1cbe8483cc41
Author: Derrick Marcey <dmarcey@github.com>
Date:   Tue May 13 05:02:48 2025 -0700

    Create a parent route for the App component

diff --git a/ui/packages/code-referencing/code-referencing.ts b/ui/packages/code-referencing/code-referencing.ts
index cce1e6a752234..586ad02dcfca3 100644
--- a/ui/packages/code-referencing/code-referencing.ts
+++ b/ui/packages/code-referencing/code-referencing.ts
@@ -19,8 +19,13 @@ registerNavigatorApp('code-referencing', () => ({
 }))

 export const codeReferencingApp = codeReferencingAppBuilder.createDataRouterAppFromRoutes([
-  codeReferencingShowRoute.toRoute({
-    Component: ShowEntrypointFuture,
-  }),
+  {
+    Component: App,
+    children: [
+      codeReferencingShowRoute.toRoute({
+        Component: ShowEntrypointFuture,
+      }),
+    ],
+  },
 ])
 registerDataRouterApp(codeReferencingApp)
+

