diff --git a/node_modules/nx/src/tasks-runner/cache.js b/node_modules/nx/src/tasks-runner/cache.js
index 496a46b..c62b17c 100644
--- a/node_modules/nx/src/tasks-runner/cache.js
+++ b/node_modules/nx/src/tasks-runner/cache.js
@@ -36,7 +36,7 @@ function dbCacheEnabled(nxJson = (0, nx_json_1.readNxJson)()) {
             process.env.NX_REJECT_UNKNOWN_LOCAL_CACHE === 'false') {
             readMoreLink += '#nxrejectunknownlocalcache';
         }
-        logger_1.logger.warn(`Nx is configured to use the legacy cache. This cache will be removed in Nx 21. Read more at ${readMoreLink}.`);
+        // logger_1.logger.warn(`Nx is configured to use the legacy cache. This cache will be removed in Nx 21. Read more at ${readMoreLink}.`);
         return false;
     }
     // ...but if on wasm and the the db cache isnt supported we shouldn't warn
diff --git a/node_modules/nx/src/tasks-runner/run-command.js b/node_modules/nx/src/tasks-runner/run-command.js
index 6fb6cfc..a135852 100644
--- a/node_modules/nx/src/tasks-runner/run-command.js
+++ b/node_modules/nx/src/tasks-runner/run-command.js
@@ -533,15 +533,15 @@ function getRunner(nxArgs, nxJson) {
     }
     const modulePath = getTasksRunnerPath(runner, nxJson);
     try {
-        if (isCustomRunnerPath(modulePath)) {
-            output_1.output.warn({
-                title: `Custom task runners will no longer be supported in Nx 21.`,
-                bodyLines: [
-                    `Use Nx Cloud or the Nx Powerpack caches instead.`,
-                    `For more information, see https://nx.dev/nx-enterprise/powerpack/custom-caching`,
-                ],
-            });
-        }
+        // if (isCustomRunnerPath(modulePath)) {
+        //     output_1.output.warn({
+        //         title: `Custom task runners will no longer be supported in Nx 21.`,
+        //         bodyLines: [
+        //             `Use Nx Cloud or the Nx Powerpack caches instead.`,
+        //             `For more information, see https://nx.dev/nx-enterprise/powerpack/custom-caching`,
+        //         ],
+        //     });
+        // }
         const tasksRunner = loadTasksRunner(modulePath);
         return {
             tasksRunner,
